src/main/resources/application.yaml

```diff
@@ -60,6 +60,10 @@ queues:
   notifications:
     connection-string: ${QUEUE_NOTIFICATIONS_SEND:"NO_VALUE_SUPPLIED"}
     queue-name: notifications
+  read-notifications:
+    connection-string: ${QUEUE_NOTIFICATIONS_READ:"NO_VALUE_SUPPLIED"}
```

Comment:
I can see this has been done by analogy to the other queue, so it's ok, but I wonder if we need a default value at all. Shouldn't the app fail to start if the env property is missing?

Reply:
in real life, if default is used - app will fail because of failure to parse the connection string. i believe this is provided so that application can be loaded in development. it also takes play in integration tests

Reply:
fair enough